crates/model/src/order.rs

```diff
@@ -693,6 +693,10 @@ pub struct OrderMetadata {
     /// Full app data that `OrderData::app_data` is a hash of. Can be None if
     /// the backend doesn't know about the full app data.
     pub full_app_data: Option<String>,
+    /// If the order was crated with a quote, then this field contains the
+    /// quote's metadata.
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub quote_metadata: Option<serde_json::Value>,
```

Comment:
Given that it's also helpful to expose the actual quoted amounts I think the format of this would better be sth like:
```
quote: {
    // .. fields stored in `order_quotes` table
}
```
That would expose all the necessary data at once.